opus/import/obs_volume_coiss_12xxx.py

```
@@ -296,32 +131,10 @@ def field_obs_type_image_lesser_pixel_size(self):
-            return _COISS_FILTER_WAVELENGTHS[key2]
-
-        self._log_nonrepeating_warning('Ignoring unknown COISS filter combination '+
-                                      f'{key[0]}/{key[1]}/{key[2]}')
-        return None, None, None
-
     def field_obs_wavelength_wavelength1(self):
         camera = self._index_col('INSTRUMENT_ID')[3]
         filter1, filter2 = self._index_col('FILTER_NAME')
-        central_wl, fwhm, effective_wl = self._coiss_wavelength_helper(
-                                                    camera, filter1, filter2)
+        central_wl, fwhm, effective_wl = self._coiss_wavelength_helper(camera, filter1, filter2)
```

Ruff (0.15.9)

[warning] Unpacked variable `effective_wl` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)